LongWarOfTheChosen/Config/XComClassData.ini

```diff
@@ -1613,3 +1613,246 @@ bHasClassMovie=true
 +bHideInCharacterPool=true
 +IconImage="img:///UILibrary_StrategyImages.MissionIcon_Resistance"
 
+
+[Skirmisher X2SoldierClassTemplate]
+!RandomAbilityDecks=()
++RandomAbilityDecks=(DeckName="SkirmisherTier1XComAbilities", \\
+					Abilities=((AbilityName="LightningReflexes_LW"), \\
+							   (AbilityName="SkirmisherReturnFire", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
+							   \\
+							   (AbilityType=(AbilityName="SkirmisherInterrupt")), \\
+							   (AbilityName="NeedleGrenades"), \\
+							  ))
++RandomAbilityDecks=(DeckName="SkirmisherTier2XComAbilities", \\
+					Abilities=((AbilityName="HeavyOrdnance_LW"), \\
+							   (AbilityName="CombatPresence"), \\
+							   (AbilityName="HeavyOrdnance_LW"), \\
+							   (AbilityName="Sapper"), \\
+							   \\
+							   (AbilityType=(AbilityName="LoneWolf")), \\
+							   (AbilityType=(AbilityName="TotalCombat")), \\
+							   (AbilityType=(AbilityName="ZeroIn", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon)), \\
```

Comment:
@jmartinez989 These entries are invalid. There should be no `AbilityType`, just `AbilityName` and `ApplyToWeaponSlot`. See other entries. You should see a redscreen on startup because of this.